source/features/unhide-reactions-on-mobile.css

```diff
@@ -0,0 +1,4 @@
+/* :has() does not work on Firefox, unless `about:config` → `layout.css.has-selector.enabled`: `true` */
+.d-none:has(.js-comment-header-reaction-button) {
```

Comment:
This can match any large item that is temporarily hidden. My selector worker. Also the Firefox-related comment isn’t necessary

Reply:
> This can match any large item that is temporarily hidden. My selector worker.

Not sure what you mean by that. This is `.d-none:has(.js-comment-header-reaction-button)`? What is any large item, and your selector worker?

Reply:
Yep, my mistake. I had it in there initially, but didn't get any results (was using stock Firefox). I tried reducing it to bare minimum components and combining them step by step.

Reply:
Sorry it was as a typo. I updated the comment. 

Your selector can affect even the outer element, making it visible when it shouldn't:

``` html
<div class="d-none comment">
  This whole comment is not meant to be visible, ever

  <div class="d-none">
    <details class="js-comment-header-reaction-button">
  </div>
</div>